### ISSUE TITLE
EUR detected as USD

### ISSUE THREAD

**Issue by dansanti** [craigchristenson/woocommerce-2checkout-api#13]
Hi! ihave a multy currency site, and, when user select eur, on my account, is detected as USD and I receive  a USD payment

**Comment by craigchristenson** [comment]
The extension uses your woocommerce shop currency on all transactions. According to the [WooCommerce documentation](https://docs.woothemes.com/document/shop-currency/), you can only price in 1 currency unless you are using a plugin. If you are using a plugin to get multi-currency support please let me know which one and I will see if I can add support for you.

**Comment by dansanti** [comment]
hi! i'm using this plugin https://wordpress.org/plugins/woocommerce-product-price-based-on-countries/

with paypal and others payment gateway, works fine.

**Comment by Jon007** [comment]
Yes, orders do have a currency, the order currency should be used rather than the shop currency for plugin compatibility.
WooCommerce Stripe gateway etc will use:
$order->get_currency()
